src/main/java/thePackmaster/cards/contentcreatorpack/OlexasCloak.java

```diff
@@ -0,0 +1,30 @@
+package thePackmaster.cards.contentcreatorpack;
+
+import com.evacipated.cardcrawl.mod.stslib.fields.cards.AbstractCard.PersistFields;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.actions.dimensiongatepack.HoardPayoffAction;
+import thePackmaster.cards.AbstractPackmasterCard;
+import thePackmaster.util.cardvars.HoardField;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class OlexasCloak extends AbstractContentCard {
+    public final static String ID = makeID("OlexasCloak");
+
+    public OlexasCloak() {
+        super(ID, -1, CardType.SKILL, CardRarity.COMMON, CardTarget.SELF);
+        baseBlock = 18;
+        PersistFields.setBaseValue(this, 2);
+        HoardField.setBaseValue(this, 3);
```

Comment:
As part of looking at this PR I double-checked the Hoard keyword. Could you update that keyword (in Cardstrings.json for DimensionGatePack) to have #b before the two costs it mentions (X and 0), so they get highlighted properly?

Reply:
Done!